src/components/ToDoItem.tsx

```diff
@@ -27,41 +28,41 @@ export const ToDoItem: React.FC<Props> = ({ todoItem }) => {
 
     return (
         <div className="item">
-            <input type="checkbox" checked={completed} onChange={() => complete(id)} />
-            {editedText ? (
-                <div>
-                    <input type="text" value={editedText} onChange={handleEditedText} />
-                    <button onClick={handleSaveClick}>
-                        <img
+            <div>
+                <input type="checkbox" checked={completed} onChange={() => complete(id)} />
+                {editedText ? (
```

Comment:
better to have separate component for editing input

I mean have a component which will contain html from line 33-46